bndtools.builder/_plugin.xml

```diff
@@ -36,8 +43,49 @@
 			<description>Marks exported packages with a '+' decoration, and
 				excluded packages with a strike-through.</description>
 		</decorator>
-	</extension>
 
+		<decorator
+			id="bndtools.componentDecorator"
+			label="Bndtools OSGi Component Decorations"
+			state="true"
+			adaptable="true"
+			lightweight="true"
+			class="org.bndtools.builder.decorator.ui.ComponentDecorator"
+			location="TOP_LEFT">
```

Comment:
Hi @gamerson, I chose the TOP_LEFT for Java Files because sometimes the TOP_RIGHT has an Icon there if the file is an Interface.

I don't mind changing it if you think it isn't a big concern.